src/mesh.cpp

```diff
@@ -410,8 +408,8 @@ scene::IMeshBuffer* cloneMeshBuffer(scene::IMeshBuffer *mesh_buffer)
 		video::S3DVertex2TCoords *v =
 			(video::S3DVertex2TCoords *) mesh_buffer->getVertices();
 		u16 *indices = mesh_buffer->getIndices();
-		scene::SMeshBufferTangents *cloned_buffer =
-			new scene::SMeshBufferTangents();
+		scene::SMeshBufferLightMap *cloned_buffer =
+			new scene::SMeshBufferLightMap();
```

Comment:
This is unrelated bugfix. Can you merge it (48a2dadc981aa142781d7eb30bfa352ac89c1930) separately?

Reply:
That's possible yes. Cherry-pick or applying the diff/patch works.
Which issue does it fix?

Reply:
Currently, calling `cloneMeshBuffer` on a buffer using `EVT_2TCOORDS` vertex type causes reads beyond its storage, thus leading to wrong data read or SEGV. That never happened only because MT doesn’t use such buffers at all, but they may be used in the future (e.g. #6820 uses them IIRC, so I had to fix that there).